guides/security/access_control.md

```diff
@@ -0,0 +1,18 @@
+# Access Control
+
+## Team-level access control
+
+Made Tech access controls reflect that:
+- We trust our team members to act responsibly, and to abide by the Acceptable Use Policy
+- We trust our team members to access appropriate sites when using the internet
+
+It is the responsibility of teams (delivery teams and business teams) to ensure that team members have the appropriate access to the information and services they need to perform their role.
+
+On delivery teams it is the Delivery Lead who should ensure that team members can access the information and services required to work on a client delivery. The Delivery Lead should also ensure that this access is revoked when a team member leaves the team, or a delivery shuts down.
+
+For business teams, it is the responsibility of the head of department to ensure that all team members have the appropriate level of access to tools and services. e.g. for the People Team, it is the responsibility of the Head of People. For the Operations Team it is the COO. 
+
+## Made Tech-wide access control
```

Comment:
I spoke with Colin yesterday and I think he has a view that we can lose this spreadsheet for now - worth having a chat with him

Reply:
Deleted